pkgs/development/python-modules/aioambient/default.nix

```diff
@@ -27,6 +28,15 @@ buildPythonPackage rec {
     hash = "sha256-ar2UGSlVukMD5EZsEn7TFfIOovaI+B3Ym+UeGo95oks=";
   };
 
+  patches = [
+    # https://github.com/bachya/aioambient/pull/295
```

Comment:
Can you please add a short comment on what this patch does

Reply:
Good call! I added a comment explaining the purpose of the patch.